### PR TITLE
French translation update

### PR DESCRIPTION
**Pull request by alganthe**
**When merged this pull request will:**
- Update the french strings and add the missing ones

I took changes from #3480 too.

### CODE REVIEW

File: addons/inventory/stringtable.xml [hunk 1]
```diff
@@ -33,7 +33,6 @@
             <Italian>Zaini</Italian>
             <Spanish>Mochilas</Spanish>
             <French>Sacs à dos</French>
-            <Portuguese>Mochilas</Portuguese>
```
Comment (esteldunedain) [inline addons/inventory/stringtable.xml]: I guess this was removed by mistake

File: addons/inventory/stringtable.xml [hunk 2]
```diff
@@ -43,7 +42,6 @@
             <Italian>Copricapi</Italian>
             <Spanish>Cascos</Spanish>
             <French>Equipement de tête</French>
-            <Portuguese>Capacetes</Portuguese>
```
Comment (esteldunedain) [inline addons/inventory/stringtable.xml]: Same here

File: addons/inventory/stringtable.xml [hunk 3]
```diff
@@ -53,7 +51,6 @@
             <Italian>Occhiali</Italian>
             <Spanish>Gafas</Spanish>
             <French>Lunettes</French>
-            <Portuguese>Óculos</Portuguese>
```
Comment (esteldunedain) [inline addons/inventory/stringtable.xml]: Here

File: addons/inventory/stringtable.xml [hunk 4]
```diff
@@ -63,7 +60,6 @@
             <Italian>Uniformi</Italian>
             <Spanish>Uniformes</Spanish>
             <French>Uniformes</French>
-            <Portuguese>Uniformes</Portuguese>
```
Comment (esteldunedain) [inline addons/inventory/stringtable.xml]: Here

File: addons/inventory/stringtable.xml [hunk 5]
```diff
@@ -72,8 +68,7 @@
             <Polish>Kamizelki</Polish>
             <Italian>Vests</Italian>
             <Spanish>Chalecos</Spanish>
-            <French>Gilets</French>
-            <Portuguese>Coletes</Portuguese>
```
Comment (esteldunedain) [inline addons/inventory/stringtable.xml]: Here

File: addons/inventory/stringtable.xml [hunk 6]
```diff
@@ -83,7 +78,6 @@
             <Italian>Granate</Italian>
             <Spanish>Granadas</Spanish>
             <French>Grenades</French>
-            <Portuguese>Granadas</Portuguese>
```
Comment (esteldunedain) [inline addons/inventory/stringtable.xml]: Here

File: addons/inventory/stringtable.xml [hunk 7]
```diff
@@ -92,8 +86,7 @@
             <Polish>Medyczne</Polish>
             <Italian>Medico</Italian>
             <Spanish>Médico</Spanish>
-            <French>Médical</French>
-            <Portuguese>Médico</Portuguese>
```
Comment (esteldunedain) [inline addons/inventory/stringtable.xml]: Here

**Comment by alganthe** [comment]
Apparently the new git diff changes still aren't perfect, going to fix this. :+1:

**Comment by alganthe** [comment]
good to go.

**Comment by esteldunedain** [comment]
Thanks

**Pull request merged**